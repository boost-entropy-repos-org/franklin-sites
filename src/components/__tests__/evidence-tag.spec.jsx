import React from 'react';
import renderer from 'react-test-renderer';

import EvidenceTag from '../evidence-tag';

describe('EvidenceTag component', () => {
  test('should render unreviewed tag', () => {
    const component = renderer
      .create(<EvidenceTag label="My evidence" title="My title" />)
      .toJSON();

    expect(component).toMatchSnapshot();
  });
  test('should render reviewed tag', () => {
    const component = renderer
      .create(<EvidenceTag label="My evidence" title="My title" reviewed />)
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
