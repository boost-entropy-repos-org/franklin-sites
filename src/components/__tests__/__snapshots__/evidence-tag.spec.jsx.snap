// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EvidenceTag component should render reviewed tag 1`] = `
<span
  className="evidence-tag evidence-tag--reviewed"
>
  <test-file-stub
    height={10}
    width={10}
  />
  <span
    className="evidence-tag__label"
    title="My title"
  >
    My evidence
  </span>
</span>
`;

exports[`EvidenceTag component should render unreviewed tag 1`] = `
<span
  className="evidence-tag evidence-tag--unreviewed"
>
  <test-file-stub
    height={10}
    width={10}
  />
  <span
    className="evidence-tag__label"
    title="My title"
  >
    My evidence
  </span>
</span>
`;
